docs/runners/local.md

```diff
@@ -15,35 +15,37 @@ Make sure that you have installed Docker compose on your system. You can find mo
 about this in the [installation](../guides/installation.md) guide.
 
 
-### Running a pipeline with the local runner
+### Running a dataset with the local runner
```

Comment:
I am not the biggest fan of "Running a dataset". Whats about "Materialize a dataset"